python/py-absl/Portfile

```diff
@@ -40,5 +39,5 @@ if {${name} ne ${subport}} {
         file attributes ${worksrcpath}/absl_py.egg-info/PKG-INFO -permissions a+r
```

Comment:
please verify whether this is still needed, hopefully upstream has fixed the permissions in the newer release and, if so, you can remove this block

Reply:
No longer needed. Thanks done.